Jenkinsfile

```diff
@@ -0,0 +1,11 @@
+pipeline {
+  agent any
```

Comment:
why `any` ?
If there will be Python agent or sth else, that not works.
Do we have defined some agents?

Reply:
hey, I was trying to make pull-request to my forked copy, accidently i made PR to original project.
Anyways I am trying to build an entire CI/CD pipeline using Jenkins. I have not attached any agents to my jenkins server.
It is running on the master itself. Any indicates the available agent.
After I complete my entire pipeline i will request anothe PR. This request can be disregarded.